packages/ui/dev/scripts/generators/generateMembers.ts

```diff
@@ -51,8 +51,34 @@ const generateMember = (override?: KnownMember) => ({
   },
 })
 
-export type MemberMock = ReturnType<typeof generateMember>
+const inviteMember = (invitedById: string) => ({
```

Comment:
Looks almost identical to generateMember; do we care enough about mocks generators to extract common code into a function?

Reply:
I'll fix that.

Reply:
And entry is a bit different

Reply:
Minor cleanup in: [`be0e88b`](https://github.com/Joystream/pioneer/pull/1466/commits/be0e88bd9e4b7db235e707c16e91a55d61b29eec).